app/services/bill-runs/two-part-tariff/fetch-bill-run-licences.service.js

```diff
@@ -81,34 +94,47 @@ async function _fetchBillRunLicences (id, filterIssues, filterLicenceHolderNumbe
     ])
     .page(page - 1, DatabaseConfig.defaultPageSize)
 
-  _applyFilters(reviewLicenceQuery, filterIssues, filterLicenceHolderNumber, filterLicenceStatus)
+  _applyFilters(reviewLicenceQuery, filterIssues, filterLicenceHolderNumber, filterLicenceStatus, filterProgress)
 
   return reviewLicenceQuery
 }
 
 function _filterIssues (filterIssues, reviewLicenceQuery) {
-  // if only a single issue is checked in the filter then a string is returned, otherwise it is an array
+  // When only one issue is selected in the filter, a string is returned; otherwise, an array is returned.
+  // The "no issues" filter can only be selected exclusively, so it will always be a string.
   if (typeof filterIssues === 'string') {
-    const lookupIssue = twoPartTariffReviewIssues[filterIssues]
-    reviewLicenceQuery.whereLike('issues', `%${lookupIssue}%`)
+    filterIssues === 'no-issues' ? _handleNoIssues(reviewLicenceQuery) : _handleSingleIssue(filterIssues, reviewLicenceQuery)
   } else {
-    // if we have got here then `issues` must be an array containing at least 2 records
-    const lookupIssues = filterIssues.map((filterIssue) => {
-      return twoPartTariffReviewIssues[filterIssue]
-    })
-
-    // the number of issues to check for in the where clause will vary depending on the number of issues checked. But
-    // there will always be at least 2
-    reviewLicenceQuery.where((builder) => {
-      builder
-        .whereLike('issues', `%${lookupIssues[0]}%`)
-      for (let i = 1; i < lookupIssues.length; i++) {
-        builder.orWhereLike('issues', `%${lookupIssues[i]}%`)
-      }
-    })
+    _handleMultipleIssues(filterIssues, reviewLicenceQuery)
   }
 }
 
+function _handleMultipleIssues (filterIssues, reviewLicenceQuery) {
+  const lookupIssues = filterIssues.map((filterIssue) => {
+    return twoPartTariffReviewIssues[filterIssue]
+  })
+
+  // Construct a query that checks for multiple issues. There will always be at least two issues to check for.
+  reviewLicenceQuery.where((builder) => {
+    builder
+      .whereLike('issues', `%${lookupIssues[0]}%`)
+    for (let i = 1; i < lookupIssues.length; i++) {
+      builder.orWhereLike('issues', `%${lookupIssues[i]}%`)
+    }
+  })
+}
+
+function _handleNoIssues (reviewLicenceQuery) {
+  // To search for no issues, check if the issues column is empty
+  reviewLicenceQuery.whereLike('issues', '')
```

Comment:
As you are just searching where the issues column is an empty string you shouldn't need to use `whereLike`
```suggestion
  reviewLicenceQuery.where('issues', '')
```